IOPool/Common/test/make_overlap_lumi_cfg.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TEST")
+
+process.source = cms.Source("EmptySource",
+                            numberEventsInLuminosityBlock = cms.untracked.uint32(5))
+
+process.thing = cms.EDProducer("ThingProducer", offsetDelta = cms.int32(1))
+
+
+process.sleep = cms.EDProducer("timestudy::SleepingProducer",
+    ivalue = cms.int32(1),
+    consumes = cms.VInputTag(),
+                               eventTimes = cms.vdouble(1.0, 0.001, 0.001))
```

Comment:
Indentation is off, e.g.
```suggestion
    eventTimes = cms.vdouble(1.0, 0.001, 0.001))
```

Reply:
fixed